invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -115,6 +115,13 @@ class SubjectL10NSchema(Schema):
     title = L10NString(data_key='title_l10n')
 
 
+class DescriptionTypeL10NSchema(Schema):
+    """Localization of description types."""
+
+    id = fields.String()
+    title = L10NString(data_key='title_l10n')
```

Comment:
This same structure is being repeated many times. See `TitleTypeL10NSchema`, `LanguageL10NSchema`... Can we create a `VocabularyL10Schema`? it should be generic enough, right?